nncf/common/graph/transformations/commands.py

```diff
@@ -103,25 +105,25 @@ class TargetPoint:
     will be applied.
     """
 
-    def __init__(self, target_type: TargetType):
+    def __init__(self, target_type: TargetType, target_node_name: NNCFNodeName):
         """
         Constructor.
 
         :param target_type: Type of the target point.
         """
         self._target_type = target_type
+        self.target_node_name = target_node_name
 
     @property
     def type(self) -> TargetType:
         return self._target_type
 
     def __eq__(self, other: Any) -> bool:
-        if self.__class__ is other.__class__:
-            return self.type == other.type
-        return False
+        return isinstance(other, TargetPoint) and \
+            self.type == other.type and self.target_node_name == other.target_node_name
 
     def __str__(self) -> str:
-        return str(self.type)
+        return f"{str(self.type)} {str(self.target_node_name)}"
```

Comment:
I think this should work.
```suggestion
        return f'{self.type} {self.target_node_name}'
```

Reply:
Done